usagov/vote-gov
# Vote-1414 -> in english inconsistencies updates

ticket -> [Vote-1414](https://bixal-projects.atlassian.net/browse/VOTE-1441)

description -> This pr is after reviewing for the site and (in English) inconsistencies and will provide updates to them to match when a link does not have a native translation to let the user know they will be going to an English site

to test:

1. pull down branch
2. start local 
3. visit local build URL `localhost:1313` and `localhost:1313/about` and use the screenshots below for the highlighted sections that should have "in English" added to them 

home page:
![english - hugo site](https://github.com/usagov/vote-gov/assets/88721460/4f6e1f08-f280-42b1-ab00-ce3d87d43a25)

about page:
<img width="755" alt="Screen Shot 2023-07-18 at 3 53 54 PM" src="https://github.com/usagov/vote-gov/assets/88721460/67a49209-4689-41bd-abec-508374c9f837">
